data/json/items/vehicle/battery.json

```diff
@@ -43,6 +43,20 @@
     "capacity": 150,
     "//": "~2.4Ah @ 12VDC. Could supply ~140 watts 10 minutes or so"
   },
+  {
+    "id": "large_storage_battery",
+    "copy-from": "storage_battery",
+    "type": "MAGAZINE",
+    "name": "large storage battery",
+    "name_plural": "large storage batteries",
+    "description": "A huge storage battery containing many lithium ion cells.  Holds a tremendous amount of energy.  Could be installed into a storage battery case for easy removal from a vehicle, or just welded straight in.",
```

Comment:
The description claims the large storage battery can be installed into a storage battery case, but I see no evidence of a "large removable storage battery" in your vehicle parts definitions.